ts/src/main/scala/ch/epfl/ts/engine/MarketRules.scala

```diff
@@ -47,13 +53,13 @@ class MarketRules extends Serializable {
                        matchExists: (Double, Double) => Boolean = alwaysTrue,
                        oldTradingPrice: Double,
                        enqueueOrElse: (Order, PartialOrderBook) => Unit): Double = {
```

Comment:
I hardly know what it does, it has been here since we started and I believe that if I tried to change something here, everything will break.